Dockerfile

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Using specific tag for now, there was some reason newer minio versions didn't work
-FROM quay.io/cloudservices/minio:RELEASE.2021-06-17T00-10-46Z.hotfix.35a0912ff as minio-examples
+# Using specific tag to avoid newer minio versions that don't currently work
+FROM quay.io/minio/minio:RELEASE.2021-06-17T00-10-46Z as minio-examples
```

Comment:
```suggestion
FROM docker.io/minio/minio:RELEASE.2021-06-17T00-10-46Z.hotfix.35a0912ff as minio-examples
```

Reply:
build/test went okay and a quick inspection shows this to be the same image as the copy we used to build upon from the quay.io/cloudservices account